lib/matplotlib/tests/test_backend_macosx.py

```diff
@@ -44,3 +45,26 @@ def new_choose_save_file(title, directory, filename):
         # Check the savefig.directory rcParam got updated because
         # we added a subdirectory "test"
         assert mpl.rcParams["savefig.directory"] == f"{tmp_path}/test"
+
+
+@pytest.mark.backend('macosx')
```

Comment:
Perhaps the same test makes sense for other frameworks too?

Reply:
Sure, I just added it to test_backends_interactive instead. We'll see if all the other backends are also implemented properly ;)